### PR TITLE
TM-1482 load custom basic steps

### PR DESCRIPTION
**Pull request by cvmocanu**

### CODE REVIEW

File: model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt [hunk 1]
```diff
@@ -3,7 +3,7 @@ package com.testerum.model.text.parts.param_meta.field
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.testerum.model.text.parts.param_meta.TypeMeta
 
-data class FieldTypeMeta (
-        @JsonProperty("name") val name: String,
-        @JsonProperty("type") val type: TypeMeta
+data class TypeMetaFieldDescriptor(
```
Comment (ionut-pruteanu) [inline model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt:6]: Should this be renamed on the frontend too? 
If you change the model name on backend make sure you update the frontend too.

Comment (cvmocanu) [reply model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt:6]: nice catch, thanks

fixed

File: testerum-backend/testerum-runner/testerum-runner-cmdline/src/main/kotlin/com/testerum/runner_cmdline/askjhfasdkjfadskj.kt [hunk 2]
```diff
@@ -1,9 +1,6 @@
 package com.testerum.runner_cmdline
 
-import org.fusesource.jansi.Ansi
-import org.fusesource.jansi.Ansi.ansi
-import org.fusesource.jansi.AnsiConsole
-import java.util.*
+import java.util.Random
 import javax.annotation.concurrent.NotThreadSafe
```
Comment (ionut-pruteanu) [inline testerum-backend/testerum-runner/testerum-runner-cmdline/src/main/kotlin/com/testerum/runner_cmdline/askjhfasdkjfadskj.kt:5]: Do you want to keep this class here? naybe a file rename?

**Review by ionut-pruteanu — APPROVED** [review]

File: model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt (hunk 1 above)
Comment (ionut-pruteanu) [inline model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt:6]: If you rename a model class you should rename it in the frontend too. Do you think this name is better?
I as abstrat as before for me.

**Review by ionut-pruteanu — APPROVED** [review]

File: model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt (hunk 1 above)
Comment (cvmocanu) [reply model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt:6]: It's not a ``TypeMeta`` (it doesn't implement that interface), and you couldn't see this difference when you compare ``FieldTypeMeta`` with ``StringTypeMeta``.

**Pull request merged**